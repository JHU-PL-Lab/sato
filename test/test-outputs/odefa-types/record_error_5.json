[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "rec" ],
          "err_match_val": "{x=zero, y=one}",
          "err_match_expected": "{z}",
          "err_match_actual": "{x, y}"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "ans = rec.z",
    "err_steps": 72
  }
]