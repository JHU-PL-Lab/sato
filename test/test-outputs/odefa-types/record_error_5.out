Type errors for:
- Input sequence  : []
- Found at clause : ans = rec.z
--------------------
* Value    : rec = {x=zero, y=one}
* Expected : {z}
* Actual   : {x, y}
1 errors found. No further control flows exist.
